polyfills/PageVisibility/polyfill-moz.js

```diff
@@ -0,0 +1,12 @@
+// Page Visibility API
+(function () {
+	document.hidden = typeof document.hidden === 'undefined' ? document.mozHidden : document.hidden;
+	document.visibilityState = typeof document.visibilityState === 'undefined' ? document.mozVisibilityState : document.visibilityState;
```

Comment:
This feature detect is unnecessary - you can assume the native properties are undefined.